app/scripts/modules/core/src/entityTag/EntityTagEditor.tsx

```diff
@@ -0,0 +1,328 @@
+        <a className="btn btn-link" onClick={this.onHide}>
+          <span className="glyphicon glyphicon-remove" />
+        </a>
+      </div>
+    );
+
+    const submitButton = (
+      <button type="submit" className="btn btn-primary" disabled={!isValid || isSubmitting}>
+        <span className="glyphicon glyphicon-ok-circle" />
+        <span>{this.props.isNew ? ' Create' : ' Update'} {tag.value.type}</span>
+      </button>
+    );
+
+    const cancelButton = <button type="button" className="btn btn-default" onClick={this.onHide}>Cancel</button>
+
+    const { TaskMonitorWrapper } = NgReact;
+
+    return (
+      <Modal show={this.state.show} onHide={this.onHide} dialogClassName="entity-tag-editor-modal">
+
+        <TaskMonitorWrapper monitor={this.state.taskMonitor} />
+
+        <Formsy.Form
+          role="form"
+          name="form"
+          className="form-horizontal"
+          onSubmit={this.upsertTag}
+          onValid={this.onValid}
+          onInvalid={this.onInvalid}
+        >
+          <Modal.Header>
+            <h3>{isNew ? 'Create' : 'Update'} {tag.value.type}</h3>
+            {closeButton}
+          </Modal.Header>
+
+          <Modal.Body>
+            <EntityTagMessage
+              message={message}
+              onMessageChanged={this.handleMessageChanged}
+            />
+
+            { ownerOptions && ownerOptions.length && (
+              <OwnerOptions
+                selectedOwner={this.state.owner}
+                ownerOptions={ownerOptions}
+                onOwnerOptionChanged={this.handleOwnerOptionChanged}
+              />
+            ) }
+          </Modal.Body>
+
+          <Modal.Footer>
+            {cancelButton}
+            {submitButton}
+          </Modal.Footer>
+        </Formsy.Form>
+
+      </Modal>
+    );
+  }
+}
+
+
+
+interface IEntityTagMessageProps {
+  message: string;
+  onMessageChanged(message: string): void;
+}
+
+@autoBindMethods
+class EntityTagMessage extends React.Component<IEntityTagMessageProps, {}> {
+  private handleTextareaChanged(event: React.FormEvent<HTMLTextAreaElement>): void {
+    this.props.onMessageChanged(event.currentTarget.value);
+  }
+
+  public render() {
+    const { message } = this.props;
+
+    return (
+      <div className="row">
+        <div className="col-md-10 col-md-offset-1">
+
+          <TextArea
+            label="Message"
+            _help={<div>Markdown is okay <HelpField id="markdown.examples"/></div>}
+            layout={BasicFieldLayout}
+            name="message"
+            required={true}
+            validationErrors={{ isDefaultRequiredValue: 'Please enter a message' }}
+            onChange={this.handleTextareaChanged}
+            value={message}
+            rows={5}
+            className="form-control input-sm"
+          />
+
+          { message && (
+            <div className="form-group preview">
+              <div className="col-md-3 sm-label-right">
+                <b>Preview</b>
```

Comment:
hah IDK, I just copied the existing markup.  I prefer &lt;strong&gt;
![screen shot 2017-05-19 at 3 01 54 pm](https://cloud.githubusercontent.com/assets/2053478/26268427/27c18158-3ca4-11e7-8e8e-f432914f2bf5.png)